scripts/log-capture

```diff
@@ -61,13 +64,23 @@ _to_log ip r
 _to_log journalctl
 
 cp /etc/resolv.conf ${OUTDIR}
-cp /tmp/*.log ${OUTDIR}
-cp /root/lorax-packages.log ${OUTDIR}
+
+if [[ -e /sys/firmware/efi ]]; then
+    _to_log ls -l /sys/firmware/efi/efivars
+fi
+
+if stat -t /tmp/*.log >/dev/null 2>&1; then
```

Comment:
You can nicer do that with test command, to stop using the >/dev/null .. redirection.

if test -e /tmp/*.log; then

Reply:
done